README.md

```diff
@@ -30,3 +30,48 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+Answer: 
+Let's go down the code and list everything we see. 
+
+if $n \leq 1$ 
+
+$3T(\frac{n}{3}) + n^5$ else 
+
+To find the complexity, we need to isolate it more. 
+
+_T_(n) = $3T(\frac{n}{3}) + n^5$
+
+$= 3(3T(\frac{\frac{n}{3}}{3}) + (\frac{n}{3})^5) + n^5$
+
+$= 9T(\frac{n}{3}) + \frac{3n^5}{3^5} + n^5$
+
+Can remove $`n^5`$ so it is easier. 
+
+$= 9T(\frac{n}{3}) + \frac{3}{3^5} \cdot n^5 + n^5$
+
+With this being shortened we can determine a sum 
+
+$= 3^iT(\frac{n}{3^i}) + \displaystyle\sum_{j=0}^{i-1}\frac{3^j}{(3^j)^5} \cdot n^5$ 
+
+I will use $i = \log_3(n)$ for substitution 
+
+$= 3^{\log_3(n)}T(\frac{n}{3^{\log_3(n)}}) + \displaystyle\sum_{j=0}^{\log_3(n)-1}\frac{3^j}{(3^j)^5} \cdot n^5$
+
+Simplify this so that _k_ is the constant. 
```

Comment:
But this isn't a constant, is it? The value changes with every term of the sum.

Reply:
I updated and my sources too